File: 0125-valid-palindrome/0125-valid-palindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        s1 = ""
        for ch in s:
            if ch.isalnum():
                s1+=ch.lower()
        st=0
        end=len(s1)-1
        while st<=end:
            if(s1[st]!=s1[end]):
                return False
            st+=1
            end-=1
        return True

File: 0125-valid-palindrome/0125-valid-palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        string s1 = "";
        for(char ch:s){
            if(isalnum(ch)){
                s1+=tolower(ch);
            }
        }
        string s2 = s1;
        reverse(s2.begin(),s2.end());
        return s1 == s2;
    }
};